internal/dev_server/manifesto.md

```diff
@@ -0,0 +1,17 @@
+# Dev Server Vision Doc
+
+This doc lays out our vision for what `dev-server` is supposed to do, and why this is the case.
+
+## Purpose of the Dev Server
+
+First and foremost, `dev-server` is meant as a replacement for local dev and CI environments. Other use cases may exist (and don't hestitate to ask if you think you have one!), but since a `dev-server` environment lacks any targeting, it is best suited to situations where there is only 1 user for the whole environment.
```

Comment:
```suggestion
First and foremost, `dev-server` is meant as a replacement for local dev and CI environments managed in the LaunchDarkly service. Other use cases may exist (and don't hesitate to ask if you think you have one!), but since a `dev-server` environment lacks any targeting, it is best suited to situations where there is only 1 user for the whole environment.
```